packages/sheets-filter-ui/src/worker/generate-filter-values.service.ts

```diff
@@ -137,3 +133,164 @@ export function getFilterByValueItems(
 
     return items;
 }
+
+export function getFilterTreeByValueItems(
```

Comment:
This should have test covered. For example, different number formats of cells.

Reply:
different types of formats are added to the filter test demo data